internal/pkg/telemetry/windows_cpu.go

```diff
@@ -0,0 +1,84 @@
+// +build windows
+
+//
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package telemetry
+
+import (
+	"math"
+	"syscall"
+	"unsafe"
+
+	"bitbucket.org/bertimus9/systemstat"
+)
+
+var (
+	modkernel32        = syscall.NewLazyDLL("kernel32.dll")
+	procGetSystemTimes = modkernel32.NewProc("GetSystemTimes")
+)
+
+func PollCpu() (cpuSnapshot CpuUsage) {
+	var idle, kernel, user FILETIME
+
+	GetSystemTimes(&idle, &kernel, &user)
+	idleFirst := idle.DwLowDateTime | (idle.DwHighDateTime << 32)
+	kernelFirst := kernel.DwLowDateTime | (kernel.DwHighDateTime << 32)
+	userFirst := user.DwLowDateTime | (user.DwHighDateTime << 32)
+
+	return CpuUsage{
+		Busy:  uint64(userFirst),               // linuxSample.Nice + linuxSample.User,
+		Idle:  uint64(idleFirst),               //linuxSample.Idle,
+		Total: uint64(kernelFirst + userFirst), // linuxSample.Total,
+	}
+
+}
+
+func AvgCpuUsage(init, final CpuUsage) (avg float64) {
+	// SimpleAverage only uses idle and total, so only copy those
+	windowsInit := systemstat.CPUSample{
+		Idle:  init.Idle,
+		Total: init.Total,
+	}
+
+	windowsFinal := systemstat.CPUSample{
+		Idle:  final.Idle,
+		Total: final.Total,
+	}
+
+	avg = systemstat.GetSimpleCPUAverage(windowsInit, windowsFinal).BusyPct

```

Comment:
I used SystemStat's average calculation because I was already importing the library for polling CPU usage. Importing it here just to use their simple mean calculation seems unnecessary.

Reply:
I totally agree with this, but since I didn't know exactly what was in that library and since is was already pulled in for linux i decided for the sake of staying consistent that I'd use it. Happy to remove it.